core/src/com/unciv/ui/utils/ImageGetter.kt

```diff
@@ -241,18 +240,20 @@ object ImageGetter {
         if (imageExists("UnitPromotionIcons/$basePromotionName")) {
             val icon = getImage("UnitPromotionIcons/$basePromotionName")
             icon.color = colorFromRGB(255, 226, 0)
-            val circle = icon.surroundWithCircle(30f)
+            val circle = icon.surroundWithCircle(size)
             circle.circle.color = colorFromRGB(0, 12, 49)
             if (level != 0) {
                 val starTable = Table().apply { defaults().pad(2f) }
-                for (i in 1..level) starTable.add(getImage("OtherIcons/Star")).size(8f)
+                for (i in 1..level) starTable.add(getImage("OtherIcons/Star")).size(size / 3f)
                 starTable.centerX(circle)
-                starTable.y = 5f
+                starTable.y = size / 6f
                 circle.addActor(starTable)
             }
             return circle
         }
-        return getImage("UnitPromotionIcons/" + promotionName.replace(' ', '_') + "_(Civ5)")
+        return Table().apply {
+            add(getImage("UnitPromotionIcons/" + promotionName.replace(' ', '_') + "_(Civ5)")).size(size)
```

Comment:
This is no longer needed, it was for a transition period where we were converting the file names.
We can just assume that the image exists and worst case, we'll get a white image